keyboards/alt34/readme.md

```diff
@@ -0,0 +1,66 @@
+# alt34
+
+![alt34](https://i.imgur.com/GDv0dL0.png)
```

Comment:
```suggestion
![alt34](https://i.imgur.com/GDv0dL0h.png)
```

Reply:
I'm not that familiar with the fine details of imgur, what does adding `h` to the end of the filename do?